indico/htdocs/sass/modules/_contributions.scss

```diff
@@ -21,3 +21,69 @@
 @import "contributions/layout";
 @import "contributions/display";
 @import "contributions/editor";
+
+
+@mixin item-picker-label() {
+    .label {
+        float: left;
+        max-width: 80%;
+        overflow: hidden !important;
+        text-overflow: ellipsis;
```

Comment:
Maybe we should wrap instead? The title attr solution is a bit dirty and not the most usable if the case is really that there are several sessions with long names but only differing at the end of the string.